.github/workflows/selfdrive_tests.yaml

```diff
@@ -301,6 +301,7 @@ jobs:
       timeout-minutes: 30
       run: |
         ${{ env.RUN }} "CI=1 coverage run selfdrive/test/process_replay/test_processes.py -j$(nproc) && \
+                        chmod -R 777 /tmp/comma_download_cache && \
```

Comment:
can we do this at the end of our custom setup action?

Reply:
that would require spinning up the docker image twice ($RUN has to be called twice), but probably isn't that big of a deal

Reply:
that should have almost no overhead. the goal is to make this bug harder to write in the future

Reply:
it also needs to run at the end because the files are created by the root user while in docker

Reply:
I can add it to the end of the docker_run script when I get around to making that